### PR TITLE
V1.4dg

### LINKED ISSUE #191: [DG] Add exercise feature has log
image

### LINKED ISSUE #192: [DG] Find feature in notes of Add log
image

### LINKED ISSUE #193: [DG] Missing use cases in Appendix

### LINKED ISSUE #194: [DG] Update non functional requirements
Non-functional requirements in Appendix not updated.

### PR DESCRIPTION
**Pull request by WhiteLio**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -399,29 +462,71 @@ testers are expected to do more *exploratory* testing.
...
+   1. Test case: `edit 1 c/no abs were hurt`<br>
+      Expected: Edits the log at index 1, with a comment of `no abs were hurt`. Details of the edited log shown in the status message.
+
+   1. Test case: `edit 1 r/20 c/no abs were hurt`<br>
+      Expected: Edits the log at index 1, with reps of `20` and a comment of `no abs were hurt`. Details of the edited log shown in the status message.
 
+   1. Test case: `edit 1`<br>
+      Expected: No log is edited. Error details shown in the status message.
+
+   1. Other incorrect edit commands to try: `edit`, `edit x ...` (where x is larger than the list size)<br>
```
Comment (Jillzyt) [inline docs/DeveloperGuide.md]: Larger than the list size or smaller than 0?

**Review by Jillzyt — DISMISSED** [review]
invalid Xs can be larger than the list size and or smaller than 0?

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (WhiteLio) [reply docs/DeveloperGuide.md]: yup should be less than 1, changed :)

**Review by Jillzyt — APPROVED** [review]
LGTM (y)

**Pull request merged**